src/libraries/System.Net.Http/tests/UnitTests/Headers/HttpRequestHeadersTest.cs

```diff
@@ -739,14 +749,15 @@ public void From_UseAddMethod_AddedValueCanBeRetrievedUsingProperty()
         [Fact]
         public void From_UseAddMethodWithInvalidValue_InvalidValueRecognized()
         {
+            // values are not validated, so invalid values are accepted
```

Comment:
But this one *can* be removed, it's calling GetParsedValues which is pointless if we don't have a parser.

Reply:
Actually, per my comments above about using TokenParser, we should keep this and use it to verify that multiple-valued headers are disallowed.